bool contains_object
uint64 activation_count
uint64 index

